draft-ietf-add-ddr.md

```diff
@@ -342,6 +340,20 @@ Resolver owners that support authenticated discovery will need to list valid
 referring IP addresses in their TLS certificates. This may pose challenges for
 resolvers with a large number of referring IP addresses.
 
+## Server Name Handling
+
+When resolvers are designated using queries for "resolver.arpa" {{bootstrapping}},
+the name of the Designated Resolver is sent via the Server Name Indication
+extension of TLS {{?RFC8446}} for both DoT and DoH, and the host in the URI for
+DoH.
+
+Clients MUST NOT use "resolver.arpa" as the server name. Clients can either use
+IP address of the designating Unencrypted Resolver or the TargetName in the
+ServiceMode SVCB record as the server name.
```

Comment:
You can't put an IP in SNI, but this reads like you might be suggesting that.

My suggestion would be that "Clients MUST NOT send a TLS SNI when establishing a connection to a server that was discovered with a query to "resolver.arpa"."